ansible/playbook.yml

```diff
@@ -206,3 +206,9 @@
       shell: |
         sudo -u ubuntu bash -c ". /nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh && nix profile remove osquery"
       when: stage2_nix
+
+    - name: nix collect garbage
+      become: yes
+      shell: |
+        sudo -u ubuntu bash -c ". /nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh && nix-collect-garbage -d"
+      when: stage2_nix
```

Comment:
Missing new line here

Reply:
Thank you fixed with https://github.com/supabase/postgres/pull/1109/commits/5ad1be53eff0a814a383f2a299a1c3321d3ff82a